packages/cli/src/WorkflowRunner.ts

```diff
@@ -462,6 +465,12 @@ export class WorkflowRunner {
 			// Normally also workflow should be supplied here but as it only used for sending
 			// data to editor-UI is not needed.
 			await hooks.executeHookFunctions('workflowExecuteBefore', []);
```

Comment:
One important thing that we need to test is that this is super important in cloud.

This is how cloud prevents execution of a workflow if user is over quota. So this code basically throws an exception and the execution is aborted. We need to make sure this behavior continues to work normally.

Reply:
this presumably refers to the note I put into the comment after this line, so to prevent further confusion i'll just remove the comment